ipaserver/install/ldapupdate.py

```diff
@@ -328,12 +329,15 @@ def _identify_arch(self):
            etc.  Determine if a suffix is needed based on the current
            architecture.
         """
-        bits = platform.architecture()[0]
-
-        if bits == "64bit":
-            return "64"
+        if NAME is 'debian':
```

Comment:
Here the guilt is kind of on our side. Starting Fedora 28, the server code only supports 64-bit architectures (https://bugzilla.redhat.com/show_bug.cgi?id=1544386) so we could return `64` on Fedora and whatever is returned on Debian, have that to be returned there.

Reply:
the problem is that what it returns is platform specific, so fedora dropping 32bit support doesn't help me at all

Reply:
err, arch specific

Reply:
Now it's been some time since I last played with `ipaplatform`, but wouln't it help if we add `LIBARCH='64'` to `ipaplatform/base/paths.py` and have the current `LIBARCH="/{0}".format(MULTIARCH)` in `ipaplatform/debian/paths.py`?
Also, is this a path or a constant? I believe it might be the latter.

Reply:
Or maybe really the other way around, have Fedora override the `"/{0}".format(MULTIARCH)` since it seems more accurate.

Reply:
something like that yes